strings/strings_uk.arb

```diff
@@ -334,5 +334,11 @@
   "threadgateViewFollowing": "Користувачі за якими стежить {username}",
   "threadgateViewList": "Користувачі зі списку \"{listName}\"",
   "settingsPanelModeration": "Модерація",
-  "settingsSound": "Обрати звук сповіщення"
-}
\ No newline at end of file
+  "settingsSound": "Обрати звук сповіщення",
+  "scheduleInvalidAccount": "Ви намагалися запланувати публікацію за допомогою облікового запису, що немає доступу до планування, будь ласка, налаштуйте його.",
+  "confirmRetryScheduledPost": "Повторити заплановану публікацію",
+  "confirmRetryScheduledPostContent": "Цю публікацію не вдалося виконати, переконайтеся, що ви належним чином привʼязали свій обліковий запис для запланованих дописів, перш ніж повторити спробу. Її буде зроблено у зазначений час, і вона зʼявиться у вашому профілі «в минулому». Ви впевнені, що хочете, повторити ця публікацію?",
```

Comment:
```suggestion
  "confirmRetryScheduledPostContent": "Цю публікацію не вдалося виконати, переконайтеся, що ви належним чином привʼязали свій обліковий запис для запланованих дописів, перш ніж повторити спробу. Її буде зроблено у зазначений час, і вона зʼявиться у вашому профілі «в минулому». Ви впевнені, що хочете, повторити цю публікацію?",
```

Reply:
Oh, did you mean to add this change? I believe you would need to open another PR since I merged this one :(